### PR TITLE
New Package: py-gsutil

### PR DESCRIPTION
**Pull request by dorton21**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-gsutil-feedstock/package.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyGsutilFeedstock(PythonPackage):
+    """A command line tool for interacting with cloud storage services."""
+
+    homepage = "https://github.com/conda-forge/gsutil-feedstock"
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-gsutil-feedstock/package.py]:
```suggestion
    homepage = "https://cloud.google.com/storage/docs/gsutil"
```

File: var/spack/repos/builtin/packages/py-gsutil-feedstock/package.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyGsutilFeedstock(PythonPackage):
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-gsutil-feedstock/package.py:9]:
```suggestion
class PyGsutil(PythonPackage):
```
Will need to update file name too.

File: var/spack/repos/builtin/packages/py-gsutil-feedstock/package.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    """A command line tool for interacting with cloud storage services."""
+
+    homepage = "https://github.com/conda-forge/gsutil-feedstock"
+    pypi     = "gsutil/gsutil-4.59.tar.gz"
+
+    maintainers = ['dorton21']
+
+    version('4.59', sha256='349e0e0b48c281659acec205917530ae57e2eb23db7220375f5add44688d3ddf')
+
+    depends_on('py-setuptools', type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-gsutil-feedstock/package.py:19]: Many dependencies missing:
```python
requires = [                                                                             
    'argcomplete>=1.9.4',                                                                
    'crcmod>=1.7',                                                                       
    'fasteners>=0.14.1',                                                                 
    'gcs-oauth2-boto-plugin>=2.7',                                                       
    'google-apitools>=0.5.30',                                                           
    'httplib2>=0.18',                                                                    
    'google-reauth>=0.1.0',                                                              
    # TODO: Sync submodule with tag referenced here once #339 is fixed in mock.          
    'mock==2.0.0',                                                                       
    'monotonic>=1.4',                                                                    
    'pyOpenSSL>=0.13',                                                                   
    'retry_decorator>=1.0.0',                                                            
    'six>=1.12.0',                                                                       
]    
...
    python_requires='>=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, <4',                 
```

**Review by adamjstewart — CHANGES_REQUESTED** [review]

**Comment by dorton21** [comment]
Package is no longer needed

**Pull request closed without merging**